apps/src/maze/animationsController.js

```diff
@@ -0,0 +1,685 @@
+const SVG_NS = require('../constants').SVG_NS;
+const drawMap = require('./drawMap');
+const displayPegman = drawMap.displayPegman;
+const getPegmanYForRow = drawMap.getPegmanYForRow;
+const timeoutList = require('../lib/util/timeoutList');
+const utils = require('../utils');
+const tiles = require('./tiles');
+
+module.exports = class AnimationsController {
+  constructor(maze, svg) {
+    this.maze = maze;
```

Comment:
Long term is the plan to only pass in the options we need, vs. the entire Maze reference?

Reply:
yep!